README.md

```diff
@@ -257,7 +257,7 @@ config = defconfig do
 end
 
 # parse a filter from map syntax
-{:ok, filter} = Filtrex.parse(config, %{
+filter_dump = %{
```

Comment:
Can we use the terminology `filter_map` instead of `filter_dump` to be consistent? I think we've landed on a better term than my original proposition of `dump`.